CONTRIBUTING.md

```diff
@@ -53,17 +53,17 @@ you can specify a test directory, file, module, class or method:
 
 ## Building binaries
 
-Linux:
+`script/build-linux` will build the Linux binary inside a Docker container:
 
     $ script/build-linux
 
-OS X:
+On OS X, you should build inside a Mountain Lion VM for proper compatibility.
```

Comment:
Might be worth pointing out that this is optional.... if you know the binary is going on only be run on Mavericks, then you can build it on Mavericks.